projects/canopy/src/lib/separator/docs/guide.stories.mdx

```diff
@@ -0,0 +1,62 @@
+import { Meta } from '@storybook/addon-docs';
+
+<Meta title="Components/Separator/Guide" parameters={{ viewMode: 'docs', previewTabs: { canvas: { hidden: true } } }} />
+
+# Separator
+
+<p class="standfirst">Separators are used to visually divide sections of content.</p>
+
+![example](docs/accordion/example.png)
+
+## Usage
+
+Use this component when you want to split your content into clear sections or groupings.
+
+Import the separator module into your module:
+
+```
+@NgModule({
+    ...
+    imports: [ ..., LgSeparatorModule ],
+})
+```
+
+
+
+And in your HTML:
+
+```
+<lg-separator [variant]="variant" [hasRole]="hasRole"></lg-separator>
+```
+
+## How it works
+
+The separator component can be positioned at either full or partial width to help define distinct sections or groups within the page, a container or body content.
+
+## Dos and Don'ts
+
+### Do
+
+![do](docs/accordion/do.png)
+
+1. Do use a separator to group items, such as within a list
```

Comment:
Could you make the Do and Don't bold please? (see other comment for reference)

Reply:
done 👍